src/pudl/transform/ferc1.py

```diff
@@ -3429,11 +3481,7 @@ def transform(
             transformed_fuel=ferc1_transformed_dfs["fuel_ferc1"].copy(),
         )
 
-    # convert types and return:
-    return {
-        name: convert_cols_dtypes(df, data_source="ferc1")
-        for name, df in ferc1_transformed_dfs.items()
-    }
+    return ferc1_transformed_dfs
```

Comment:
This is also a leftover from before we created the `enforce_schema()` step and integrated it into the parent transformer class, which made it redundant.